### ISSUE TITLE
Windows 10 version 10.0.18363.657 Now Working.

### ISSUE THREAD

**Issue by stvkodra** [stascorp/rdpwrap#988]
Replace the rdpwrap.ini file in c:\Program Files\RDP Wrapper with the one provided.
[rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4193636/rdpwrap.zip)

**Comment by EspacioCivico** [comment]
Thank you very much Stvkodra this if it worked for me, just replace with the one you share and that's it. I send you a hug

**Comment by ChrDav135** [comment]
Yep. Working for me.
Thanks very much stvkodra.

**Comment by itsAllDigital** [comment]
Replaced the ini file and the config tool says it's installed and fully supported but if I try it won't let another user connect without the first one disconnecting.

Windows Version is (10.0.18362.657)

**Comment by strarsis** [comment]
It works!

**Comment by jul974** [comment]
Hello,
It looks like the "wrapp updater" is not aware of that new .ini
Do you think it was released to the automatic updater ?
Can anyone to that ? (As I have no idea of how to do this)

**Comment by Korvinho** [comment]
Winver  - 18363.657  - work !

**Comment by VelociRacer** [comment]
I have a 32-bit system and the above rdpwrap file only shows support for x64. Now Listener State is [Fully supported] but Not listening. Can someone please help? Is having 32-bit my problem? Does the rdpwrapper need to be updated to include 32-bit? Maybe the subject is a typo but I have 10.0.18362.657.

**Comment by a-lascu** [comment]
I am in the same situation as VeloiRacer. I have a Windows 10 x32 (18363.657). Can someone help us?

**Comment by stvkodra** [comment]
Try this file for x86 machines. No guarantee it will work.
[rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4200776/rdpwrap.zip)

**Comment by a-lascu** [comment]
Thank you, stvkodra.
Unfortunately it's not working.
My Win10 x32 is 18363.657 version.
![Annotation 2020-02-13 224629](https://user-images.githubusercontent.com/49077001/74476759-b4a3cb80-4eb2-11ea-80e6-0413ceb3546d.jpg)

**Comment by VelociRacer** [comment]
Thanks so much for the quick support. I tried the above rdpwrap.ini but still show Not listening [Fully supported]

Is it necessary to reboot? Or just swap out the file and run RDPConf to check for changes?

Also, I noticed the x64 lines are duplicated now in the ini file. May need to re-look at that.

**Comment by darkvampire1** [comment]
For me it was need **restart windows**. When status was ok and checker returns bad permissions.

**Comment by xrbh70** [comment]
Hi @stvkodra It work perfectly, Thanks
![imagen](https://user-images.githubusercontent.com/61043895/74513192-a4e6b000-4ecf-11ea-82b0-7714560d80bb.png)

**Comment by a-lascu** [comment]
xrbh70, your win10 is x32 or x64?
I understand that it's working ok on x64 version but I have a x32 version.
And I reboot the system after copy the new rdpwrap.ini

**Comment by aileenka** [comment]
Everything seems to be set correctly, but it is not working :( 
I have tried rebooting after replacing the .ini file, but it did not help.

My Windows version is 18363.657 & and I have used the file from stvkodra.
![rdp](https://user-images.githubusercontent.com/17024875/74518913-f5343100-4f14-11ea-8cc5-89862a10b2c1.PNG)

Anyone with similar problem?

**Comment by aileenka** [comment]
> Replaced the ini file and the config tool says it's installed and fully supported but if I try it won't let another user connect without the first one disconnecting.
> 
> Windows Version is (10.0.18362.657)

I had the same problem with Windows version 18363.567. 
Then I realized I did not stop the RDP service before replacing the .ini file.
Now it's working for me.

**Comment by a-lascu** [comment]
Still didn't work for Win10 x32 :(

**Comment by maantao** [comment]
While 64-bit works like a charm for multiple (different) user concurrent remote access, it appears that multiple sessions for same user is now broken / blocked. Appreciate if someone can confirm.

**Comment by darkvampire1** [comment]
Try set options "default RDP authentication" ,Applay, reboot

**Comment by darkvampire1** [comment]
> Thank you, stvkodra.
> Unfortunately it's not working.
> My Win10 x32 is 18363.657 version.
> ![Annotation 2020-02-13 224629](https://user-images.githubusercontent.com/49077001/74476759-b4a3cb80-4eb2-11ea-80e6-0413ceb3546d.jpg)

CFG file must be in default path! c:/Programfile/RDP-Wrapper/
Not else!

**Comment by a-lascu** [comment]
I think the problem is in the last .ini file that stvkodra have posted.
As VelociRacer noticed there are some attributes duplicates for .x86 as you can see below.
![Annotation 2020-02-14 214629](https://user-images.githubusercontent.com/49077001/74562614-e466d800-4f73-11ea-9511-747e88ed7cb3.jpg)

**Comment by VelociRacer** [comment]
Very good. Not only that, but the last block of variables above should be circled in yellow as they x64 duplicates. Who's the boss who's going to clean up the ini file? ;)
________________________________

**Comment by Kbotz** [comment]
> 
> 
> While 64-bit works like a charm for multiple (different) user concurrent remote access, it appears that multiple sessions for same user is now broken / blocked. Appreciate if someone can confirm.

Multiple sessions for same user are working here - for both W7x64 and W10x64. 

Big ups to stvkodra for the updated file.

**Comment by a-lascu** [comment]
Nobody can help us with .ini for Win10 x32?

**Comment by maantao** [comment]
> While 64-bit works like a charm for multiple (different) user concurrent remote access, it appears that multiple sessions for same user is now broken / blocked. Appreciate if someone can confirm.
> 
> Multiple sessions for same user are working here - for both W7x64 and W10x64.
> Big ups to stvkodra for the updated file.

Seems like we still face an issue even with RDP Authentication AND latest stvkodra INI

![image](https://user-images.githubusercontent.com/61052209/74640815-288fed80-51ab-11ea-9fa4-b57bc62bc0e1.png)

Are we missing the obvious for concurrent same user access?

**Comment by sanchomdv** [comment]
This worked for me in win10 10.0.18362.657 32bits x86
https://github.com/stascorp/rdpwrap/issues/999#issuecomment-586757085

**Comment by Iskandare** [comment]
@ stvkodra  Thank you , you re the best !  work perfectly for me on my 3 computer in version v 10.0.18363.657  on 64 bits

**Comment by Kbotz** [comment]
> Seems like we still face an issue even with RDP Authentication AND latest stvkodra INI
> 
> Are we missing the obvious for concurrent same user access?

Maantao, a few things you can try or look further into: 

First and foremost, don't forget the need for a single mandatory blank line at the end of your ini file and all Remote Desktop related services are started in Services.msc.

In the Session Shadowing Mode Section of the config file, maybe try "Full access without permission" or "Full access with user's permission".

Likewise in the Authentication Mode section, you could also try "GUI Authentication Only".

It could be a password settings related issue. Perhaps a Group Policy setting tweak will help, as per Local Group Policy Editor. Focus on settings in the **Connections** and **Security** subsections there. Just keep track of what you're changing so you can easily undo it if and as necessary:

**Computer Configuration\Administrative Templates\Windows Components\Remote Desktop Services\Remote Desktop Session Host\**

Enabling "Always prompt for password upon connection" in GPE could possibly do the trick also. Or the opposite. Plus how you're setting up User name and Password in the General > Logon settings in the RDP connection itself.

Just some things you could kick around regarding this issue. Wish I could offer you more, or at least something more specific. It goes without saying you're probably going to have to experiment a bit to find the solution. But those are the things I would start with as any one might do the trick. Of course someone else might come along and deliver a magic bullet. But until then, those are my best educated guesses as it relates to the problem you're experiencing.

**Comment by VelociRacer** [comment]
Great thanks for sharing, I have the same OS and 100% in business with new .ini.
________________________________

**Comment by maantao** [comment]
> Enabling "Always prompt for password upon connection" in GPE could possibly do the trick also. 

WOW Thanks Kbotz - you did provide us with the "silver" bullet amongst ALL the possible variations.

We had previously noted that saving the user password during logon of the RDP connection would cause this issue to manifest itself. However post upgrade of ini (due to Windows Update) abstaining from saving the user password didn't seem to work.

BUT forcing the password request via Group Policy Editor did the trick. Not EXACTLY sure why saving the credentials causes the login to falter, other that requiring an EXPLICIT confirmation for a same user login.

Many Thanks to all who scratched their head on this one.

**Comment by Kbotz** [comment]
> WOW Thanks Kbotz - you did provide us with the "silver" bullet amongst ALL the possible variations.

You did yourselves the biggest favor - you dug in and tried things out. And the conclusion you've drawn sounds logical. Plus there were lots of possible variations to be tested so you did well. All I did was provide a few pointers and clues to help get the ball rolling.

Thanks for the feedback. I've helped a few others in the past to boot, and been helped in return. Looking out for one another helps keep these kinds of communities strong. Happy RDPW trails.

**Comment by StarfighterJ** [comment]
Thanks stvkodra that did work but what happened to the Autoupdater? is it broken?

**Comment by Matheus-Garbelini** [comment]
Hi, it's working for 10.0.18362.657 but only for administrator.
Is there a way to make it work with a local user account?

**Comment by Kbotz** [comment]
> 
> 
> Hi, it's working for 10.0.18362.657 but only for administrator.
> Is there a way to make it work with a local user account?

This is a permissions issue and inherent to RDP. Here's a few links to help you around it:

[Allow Non-Administrator Users to Access Windows 7 Home Premium Via Remote Desktop](https://superuser.com/questions/680572/allow-non-administrator-users-to-access-windows-7-home-premium-via-remote-desktop)

[How to Allow Standard User to RDP/Remote Access on Windows 7 and 10](https://appuals.com/how-to-allow-standard-user-to-rdp-remote-access-on-windows-7-and-10)

[Cannot Establish Remote Desktop Connection Using Non-Administrator Account](https://forums.tomshardware.com/faq/cannot-establish-remote-desktop-connection-using-non-administrator-account.1505659)

**Comment by Matheus-Garbelini** [comment]
@Kbotz Thanks, I've followed the tutorial which required adding a new register and worked fine.

**Comment by Kbotz** [comment]
@**Matheus** We'll call it different strokes for different folks. Glad it worked out.

**Comment by saadalvi92** [comment]
![Screenshot (1)](https://user-images.githubusercontent.com/53086047/74866708-195c9b80-5375-11ea-9bf2-addc6e27ac0a.png)
tried every thing copied the latest file but still it's not working
any help!!!!!!!!

**Comment by VelociRacer** [comment]
Try changing port back to 3389
________________________________

**Comment by Kbotz** [comment]
> 
> 
> tried every thing copied the latest file but still it's not working
> any help!!!!!!!!

Are you on 32-bit or x64? VelociRacer is right. Having no background history specific to your setup or what RDPW experience you may have, it might be better if you uninstall RDPW and start anew with a clean working baseline, then move forward from there. For example, if you've hardened your system in any way to improve security via gpedit or the registry, revert those changes back to their basic, default settings. Also temporarily turn off your firewall and anti-virus. Then after RDP Wrapper is uninstalled, make sure you can connect with a basic RDP handshake on default port 3389. Once that is established, redeploy RDP Wrapper -- again advisably on the default RDP port to keep things as simple as possible.

When reinstalling the utility, make sure you're doing so with the administrative privileges, to include all commands you run (right click, run as administrator). Run _net stop termservice_ in cmd.exe. Then append the new (applicable) code to the end of the ini config file, and ensure this ini file is indeed in the correct programs files directory. Also be sure you are leaving an empty line at the end of the text. Run _net start termservice_ in cmd.exe afterwards, or RDPWInst.exe via cmd.exe _RDPWInst -r_ . Then reboot your system. Next run RDPCheck to confirm the handshake process is working correctly, then RDPConf to validate this further while choosing your preferred interface options. 

Consider trying a lighter authentication (security) mode in the configuration interface to see if that might help also (GUI only). Only after this is done and everything is working soundly (RDPW interface reflects all green) would I re-enable your firewall and anti-virus, custom configure your RDP port and harden connection security as desired. Harnessing a simple back to basics approach, while stepping things thru carefully, is often all it takes (and the difference between seeing red and green).  🌞

**Comment by psycmos** [comment]
Works Great for me too! X64

**Comment by fasteddieh** [comment]
Thank you very much @stvkodra this work for both of my remote servers.  Wow.

**Comment by ArtyGator** [comment]
I tried everything but it's not working, listener's OK but it still says "not supported".
(Win 10 pro x64)

EDIT: I finnally solved this by putting the file in the correct Program Files directory. Thanks to @Kbotz for giving a helpful explanation.

**Comment by XelaUA** [comment]
It's works!!! Thnak you!

**Comment by Qroxyquan** [comment]
It's works!!! Thnak you!

**Comment by Driegos** [comment]
thanks bro

**Comment by isedaniel** [comment]
It works perfectly! Thanks!!!

**Comment by Lotus-Blue** [comment]
> Replace the rdpwrap.ini file in c:\Program Files\RDP Wrapper with the one provided.
> [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4193636/rdpwrap.zip)

Thank you very much Stvkodra this if it worked for me, just replace with the one you share and that's it. I send you a hug

**Comment by maestro2369** [comment]
Big Up !! It works ! Thanks a lot :)

**Comment by actanner** [comment]
Is there ANY chance that an ini file like this exists for an X86 machine?   I have one lonely such computer which could benefit from an update.   Thanks!!

**Comment by actanner** [comment]
My apologies to the group.  I didn't read the entire thread.  And thanks to sachmomdv for pointing me to an .ini file for 18362.657  x32 that works !

**Comment by affinityv** [comment]
Updated [rdpwrap.ini](https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini)  file with sections for 10.0.18362.657 which cover both x86 and x64 now.

**Comment by wildwind122** [comment]
It works, thanks a lot !

**Comment by kenneth104** [comment]
Thanks it's works~~~~~

**Comment by Visors** [comment]
win10 18363.657
it works, thx a lot

**Comment by 700nader007** [comment]
![RDP2020working](https://user-images.githubusercontent.com/62025576/76335794-19154780-6306-11ea-90df-3f775a8d4ab1.jpg)
 latest Windows 10 update (1909) and all green and working 100%
[RDPWrap-v1.6.2-Win10pro 18363.657.zip](https://github.com/stascorp/rdpwrap/files/4313589/RDPWrap-v1.6.2-Win10pro.18363.657.zip)

**Comment by affinityv** [comment]
Make sure your INI file is good, blank line at the end and otherwise intact.

**Comment by thanhtam23** [comment]
mine still not work, pls help
![image](https://user-images.githubusercontent.com/62202340/76699222-674f8f80-66e6-11ea-852e-554b723d6288.png)

**Comment by sicwolf** [comment]
> Replace the rdpwrap.ini file in c:\Program Files\RDP Wrapper with the one provided.
> [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4193636/rdpwrap.zip)

Thanks for your information!
It works for me.

**Comment by CmdLnInt** [comment]
> Replace the rdpwrap.ini file in c:\Program Files\RDP Wrapper with the one provided.
> [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4193636/rdpwrap.zip)

I need to do this remotely ie I cannot stop terminal service. Is there some other way to update the ini?

**Comment by thanhtam23** [comment]
> > Replace the rdpwrap.ini file in c:\Program Files\RDP Wrapper with the one provided.
> > [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4193636/rdpwrap.zip)
> 
> I need to do this remotely ie I cannot stop terminal service. Is there some other way to update the ini?

you can replace the ini file directly then just restart will do

**Comment by Iskandare** [comment]
command prompt : net stop termservice, replace rdpwrap.ini and after  net start termservice

**Comment by CmdLnInt** [comment]
> > > Replace the rdpwrap.ini file in c:\Program Files\RDP Wrapper with the one provided.
> > > [rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4193636/rdpwrap.zip)
> > 
> > 
> > I need to do this remotely ie I cannot stop terminal service. Is there some other way to update the ini?
> 
> you can replace the ini file directly then just restart will do

The file is open so I cannot replace it

**Comment by CmdLnInt** [comment]
> command prompt : net stop termservice, replace rdpwrap.ini and after net start termservice

I am already connected remotely. I suppose I could try this in a batch file or powershell script.

**Comment by ioan321** [comment]
Works for me. Ur an amazing man.

**Comment by santomalo** [comment]
hi, does anyone have the new .ini for the new
 version 10.0.18362.1082

**Comment by affinityv** [comment]
@santomalo Please create a new link and provide the termsrv.dll as well as the version that `RDPConf.exe` shows for it.

**Comment by affinityv** [comment]
Okay, if it is the same as per this [issue](https://github.com/stascorp/rdpwrap/issues/1173), then it is covered with this [INI](https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini).

**Comment by sebaxakerhtc** [comment]
Duplicate of https://github.com/stascorp/rdpwrap/issues/983
